docs/file-structure.md

```diff
@@ -1,30 +1,20 @@
 # Directory Structure and File Name
 
-The [Climate Change Knowledge Portal (CCKP)](https://climateknowledgeportal.worldbank.org) is available as [**netCDF**](https://www.unidata.ucar.edu/software/netcdf) files. A netCDF4 file is structured in a hierarchical directory-like format. It consists of groups, dimensions, variables, and attributes. At the top level, there are groups that act as containers to organize data hierarchically.
+The [Climate Change Knowledge Portal (CCKP)](https://climateknowledgeportal.worldbank.org) is available as [**netCDF**](https://www.unidata.ucar.edu/software/netcdf) files.
+A netCDF4 file is structured in a hierarchical directory-like format.
+It consists of groups, dimensions, variables, and attributes. At the top level, there are groups that act as containers to organize data hierarchically.
+The CCKP netCDF data structures are described for each collection in the following sections.
 
-The **CCKP netCDF data** directory structure is as follows:
+| COLLECTION LABEL   | FILE STRUCTURE | EXAMPLE |
+|--------------------|----------------|---------|
+| CMIP6 0.25-degree  | [](./collections/cmip6-x0.25.md) | [](../notebooks/cmip6-x0.25.ipynb) |
+| CRU 0.5-degree     | [](./collections/cru-x0.5.md) | [](../notebooks/cru-x0.5.ipynb) |
+| ERA5 0.5-degree    | [](./collections/era5-x0.5.md) | [](../notebooks/era5-x0.5.ipynb) |
+| Population and Poverty | [](./collections/pop-x1.md) | [](../notebooks/pop-x1.ipynb) |
 
-> <font color='#c4c43b'>{collection}</font>/<font color='#c43bc4'>{variable}</font>/<font color='#3bc43b'>{model-run}</font>/{file}.nc
 
-## File Name
```

Comment:
@elbeejay The example color-coded filename was removed. Wasn't it helpful?

Reply:
I thought it was nice when all of the subsequent documentation was below and followed the same color scheme. Now that each collection has it's own page I could go either way - if you like it I'm happy to keep it in. Let me know.

Reply:
@elbeejay I was just curious and that makes sense. Thanks!